### ISSUE TITLE
TensorFlow binding version availability

### ISSUE THREAD

**Issue by lucaro** [silvanheller/cineast#60]
I just checked and there is still no platform specific package for windows available with a version beyond 0.8, even though they released a snapshot of 0.10 already. To get the project to compile at all on windows, the gradle build file has to be modified like this:

```
if(osdetector.os.equals('windows')){
	compile group: 'org.bytedeco.javacpp-presets', name: 'tensorflow', version: '0.8.0-1.2'
	compile group: 'org.bytedeco.javacpp-presets', name: 'tensorflow', version: '0.8.0-1.2', classifier: osdetector.classifier
}else{
	compile group: 'org.bytedeco.javacpp-presets', name: 'tensorflow', version: '0.9.0-1.2'
	compile group: 'org.bytedeco.javacpp-presets', name: 'tensorflow', version: '0.9.0-1.2', classifier: osdetector.classifier
}
```

**Comment by silvanheller** [comment]
That's a good point. We will have to do the same for mac as the osdectector doesn't fully work on macs. We have done this manually but that script looks way better.

**Comment by lucaro** [comment]
At least for macosx they bothered to release the binding for v0.9: https://repo1.maven.org/maven2/org/bytedeco/javacpp-presets/tensorflow/0.9.0-1.2/

**Comment by silvanheller** [comment]
Official comments from the devs: https://github.com/bytedeco/javacpp-presets/issues/284

**Comment by silvanheller** [comment]
Windows fix in 1369feda4f16606d5e85a7e184fafaa346cd5151
@s3inlc Would you mind Adjusting the .gradle-file in a similar way for your mac-specific issue?

**Comment by silvanheller** [comment]
Fixed
